dart/lib/vm_service_lib.dart

```diff
@@ -171,8 +171,12 @@ class VmService {
   /// [id].
   ///
   /// The return value can be one of [Obj] or [Sentinel].
-  Future<dynamic> getObject(String isolateId, String objectId) {
-    return _call('getObject', {'isolateId': isolateId, 'objectId': objectId});
+  Future<dynamic> getObject(String isolateId, String objectId,
+      [int offset, int count]) {
```

Comment:
nit: It would be nice if the generator pulled the spec documentation about these two optional arguments and added it to the dartdoc for this method.